ci/travis/dump-reports.sh

```diff
@@ -46,13 +43,10 @@ export -f dump_report
 # the two things that produce them. Note that the Clang static analysis reports
 # are copied into the scan-build-output directory by the build-docker.sh script.
 
-find "${BUILD_OUTPUT}" -name 'compat_report.html' -print0 2>/dev/null | \
-    xargs -0 -I{} bash -c 'dump_report "{}"'
-
-find scan-build-output/ -name '*.html' -print0 2>/dev/null | \
-    xargs -0 -I{} bash -c 'dump_report "{}"'
+find "${BUILD_OUTPUT}" -name 'compat_report.html' \
+    -exec bash -c 'dump_report "{}"' \; 2>/dev/null || \
+  echo "No ABI compability reports found."
 
-if [[ "${HAS_REPORTS}" = 0 ]]; then
-  echo "No analysis reports found, exit scripts with success."
-  exit 0
-fi
+find scan-build-output/ -name 'compat_report.html' \
```

Comment:
I think you accidentally changed `*.html` to `compat_report`.

Reply:
Doh! Thanks!